.gitlab-ci.yml

```diff
@@ -524,7 +524,16 @@ publish-gh-doc:
   after_script:
     - rm -vrf ${HOME}/.gitconfig
 
-
+publish-draft-release:
+  stage:                           publish
+  image:                           parity/tools:latest
+  only:
+    - tags
+    - /^v[0-9]+\.[0-9]+\.[0-9]+.*$/
```

Comment:
If I am reading this right, it would not match in `v2.0.0-alpha.1`...

Reply:
It should. The `.*` at the end will match any arbitrary string after `v#.#.#`